mvc/src/main/java/nextstep/mvc/HandlerAdapterRegistry.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.mvc;
+
+import jakarta.servlet.ServletException;
+import java.util.List;
+
+public class HandlerAdapterRegistry {
+
+    private final List<HandlerAdapter> handlerAdapters;
+
+    public HandlerAdapterRegistry(final List<HandlerAdapter> handlerAdapters) {
+        this.handlerAdapters = handlerAdapters;
+    }
```

Comment:
매번 빈 리스트를 생성하여 전달하는 것보다는 내부적으로 생성하는 것은 어떠할까요? 테스트와 프로덕션 코드에서 new ArrayList<>()를 생성하는 작업이 중복되고 있는데, 필요한 중복인지 약간 의문이 들어요. 이미 handlerAdapter들이 담겨있는 리스트가 전달되는 경우가 있으려나요 
그리고 예를 들어 unmodifiableList가 전달되는 경우, `addHandlerAdapter` 메서드를 호출했을 때 오히려 예외가 발생할 수 있을 것 같아요.

```suggestion
    private final List<HandlerAdapter> handlerAdapters = new ArrayList<>();
```

Reply:
내부에서 생성하면 테스트할 때 더 간편할 것 같네요 ㅎㅎ 반영했습니다.